semantic/src/site/globals/site.overrides

```diff
@@ -2632,12 +2633,13 @@ h6:first-child {
 
   #menu-bar {
     font-size: inherit;
+    background-color: inherit;
     .list {
       z-index: 2000;
       position: absolute;
       top: @mobileHeaderHeight;
       left: 0px;
-      background: @darkGrey;
+      background-color: inherit;
```

Comment:
Just had to make this `inherit` all the way down from "user-area" to make sure it has the same colour as the main header.